bpython/test/test_import_not_cyclical.py

```diff
@@ -71,7 +71,7 @@ def test_simple_symbolic_link_loop(self):
                 self.filepaths.remove(thing)
 
 else:
-    @unittest.skip()
+    @unittest.skip("test doesn't work in python 2")
```

Comment:
This should be something like [skipIf](https://docs.python.org/3/library/unittest.html#unittest.skipIf) with a condition that is true for Python 2.